### ISSUE TITLE
Feature request: Upcoming Events

### ISSUE THREAD

**Issue by wolovim** [Developer-DAO/academy#217]
Would love to see D_D hackathons and related opportunities featured on the site somewhere eventually. Feels like a natural part of a funnel to help learners skill up, then encourage them to go out and flex those new muscles.

**Comment by Markkos89** [comment]
@wolovim does this information about upcoming events exists in a database/cms/stored somewhere? do you know mate? thanks!

**Comment by wolovim** [comment]
not yet afaik, but perhaps it will after the dao website redesign, cc/ @kempsterrrr

**Comment by kempsterrrr** [comment]
@wolovim we're planning to include upcoming events on the main website - interested to know if we feel having events on both site makes sense - to me I'm not sure it does. Maybe a link to the main websites event page?

cc @Markkos89

**Comment by Markkos89** [comment]
> @wolovim we're planning to include upcoming events on the main website - interested to know if we feel having events on both site makes sense - to me I'm not sure it does. Maybe a link to the main websites event page?
> 
> cc @Markkos89

What about having a shared "EventsBanner" component that can work on both pages? Adaptable to different spaces of each page maybe, but I found interesting and valuable to show more this type of DAO info!! :muscle:

**Comment by kempsterrrr** [comment]
> What about having a shared "EventsBanner" component that can work on both pages? Adaptable to different spaces of each page maybe, but I found interesting and valuable to show more this type of DAO info!! 💪

Could see a component making sense, can't currently imagine based on @uma-l designs where that would live...

Feels a little bit like feature if I'm honest to aim to get it in fast but that's just me 🙃

**Comment by Markkos89** [comment]
> > What about having a shared "EventsBanner" component that can work on both pages? Adaptable to different spaces of each page maybe, but I found interesting and valuable to show more this type of DAO info!! muscle
> 
> Could see a component making sense, can't currently imagine based on @uma-l designs where that would live...
> 
> Feels a little bit like "feature creep" if I'm honest to aim to get it in fast but that's just me upside_down_face

As for now it can be a new Component uma can design also! But in the actual version. 
Then uma can include that banner or "way to show DAO's events" in the new designs!

**Comment by elPiablo** [comment]
Just want to get this out there forks. 
Regarding the above.
As long as these suggestions are not distracting a learner when in a lesson. Or a 'train of thought' in the context of a lesson e.g. if they need to switch from a project lesson to a fundamental to find some information/concepts, and then back to the project, they shouldn't be bombarded with distractions and rabbit holes. 
A lesson is a place where they can focus, and not a place for banners, bells and whistles.

**Comment by kempsterrrr** [comment]
This point from @elPiablo I think is really important - what value from a learning perspective does this provide users? How to does it help the core goal here which is helping them learn something?

> e.g. if they need to switch from a project lesson to a fundamental to find some information/concepts, and then back to the project, they shouldn't be bombarded with distractions and rabbit holes.
A lesson is a place where they can focus, and not a place for banners, bells and whistles.

Whilst the idea of adding events sounds cool I think above should be carefully considered potential issues... we don't want people not completing lessons because they're distracted by an event.

**Comment by okhaimie-dev** [comment]
> As long as these suggestions are not distracting a learner when in a lesson.

I see this being included on the landing page or a dedicated page for events = shouldn't be a distraction while learning, but I get your point. And I am sure there should be a way we can structure the user journey to avoid this outcome.

**Comment by wolovim** [comment]
yeah, what @okhaimie-dev said. agree that lessons are distraction-free zones.

besides a dedicated events page, another opportunity to surface hackathons, etc. is after a user has completed a track. e.g., give them a big congratulations and encourage them to flex those skills in the "real world," assuring them they've learned enough to give it a go.

just logging this as a nice-to-have; not expected in next iteration of MVP.

**Comment by elPiablo** [comment]
> after a user has completed a track. e.g., give them a big congratulations and encourage them to flex those skills in the "real world," assuring them they've learned enough to give it a go.

Thanks for bringing in `the track` aspect and the reward afterwards. Perfect.
